docs/.vitepress/contributors.ts

```diff
@@ -143,8 +156,17 @@ const plainTeamMembers: CoreTeam[] = [
     title: 'Frontend Developer',
     desc: 'Loves Svelte, Vite and open source, author of the UnoCSS tutorial',
   },
+  {
+    avatar: 'https://github.com/hannoeru.png',
+    name: 'ハン / Han',
+    github: 'hannoeru',
+    x: 'hannoeru',
+    bluesky: 'hannoeru.me',
+    title: 'Software Engineer',
+  },
```

Comment:
Please feel free to complete your personal information. @hannoeru

Reply:
LGTM Thanks!